tests/integration/test_ingress_relation.py

```diff
@@ -55,13 +64,16 @@ def start_server(self):
         deploy_any_charm(json.dumps(src_overwrite)),
         build_and_deploy_ingress(),
     )
+
     await ingress.set_config({"service-hostname": "any"})
     await model.wait_for_idle()
-    await run_action("any", "rpc", method="start_server")
     await model.add_relation("any:ingress", "ingress:ingress")
-    await model.wait_for_idle()
+    await model.wait_for_idle(status="active")
```

Comment:
Up to you, but may I suggest `wait_for_active=True`?

Reply:
i don't have any strong feelings, is `wait_for_active` more conventional?

Reply:
using `status=active` because `wait_for_active=True` is being flagged as deprecated, and `status=active` is the suggested way.